src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/components/step_time_field/__jest__/__snapshots__/step_time_field.test.js.snap

```diff
@@ -31,7 +31,7 @@ exports[`StepTimeField should render "Custom index pattern ID already exists" wh
   />
   <TimeField
     fetchTimeFields={[Function]}
-    isLoading={true}
+    isLoading={false}
```

Comment:
@lizozom This is the last remaining question for me. I spent some time looking at it today and I'm not sure why exactly the snapshot value has changed. 

On the one hand, @ppisljar made updates to the `indexPatternsService` mock in the test, and also to `fetchTimeFields` in the component that's being tested, so it's possible it has something to do with that.

On the other hand, it makes sense to me that `isLoading={false}` would actually be the correct value in the snapshot since this is all kicked off in the `componentDidMount` lifecycle method, so I also wonder if there was an issue with the previous implementation.

I will continue looking at this and testing the UI, but wanted to get the remaining changes pushed up so we can get this merged. @ppisljar let me know if you have any other ideas on this.